2-mandatory/4-restaurant.js

```diff
@@ -20,7 +20,18 @@ const MENU = {
 };
 
 let cashRegister = {
-  // write code here
+  orderBurger: function(balance) {
+    this.balance = balance;    
+    if (balance >= MENU.burger) {
+        return balance = balance - MENU.burger;
+    }  else return balance; 
+  },
+  orderFalafel: function(balance) {
+    this.balance = balance;
+    if (balance >= MENU.falafel) {
+        return balance = balance - MENU.falafel;
+    }  else return balance; 
+  }
 }
 
```

Comment:
I have used a similar approach with small differences, the example has declared a variable isBalanceBigEnough that calculates the balance and passes the variable into the if statement, I have passed the calculation into the if statement without assigning it to a variable